src/main/java/com/jmb/springfactory/dao/group/GroupRepository.java

```diff
@@ -1,6 +1,7 @@
 package com.jmb.springfactory.dao.group;
 
 
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)